backend/typescript/seeders/2022.10.20T21.57.52.sample-creators.ts

```diff
@@ -57,6 +57,63 @@ const seedCreators: CreatorSeed[] = [
     availability: ["TODO"],
     book_covers: ["TODO1", "TODO2"], // array of URLs
     isReadyForReview: true,
+    presentations: [
```

Comment:
Hey Caitlin, if you're having trouble with seeding the presentations you can try inserting a json array of presentations into a pre-existing creator using Postgres.

`docker exec -it ccbc-db-1 bash`
`psql -U postgres -d ccbc`
`UPDATE creator SET publications = array['{"title":"The Cat in the Hat","publisher":"Random House", "publication_year":"1957","notes":"One of the best-selling books of all time"}'::json] WHERE id=1;`

In your case you would, swap out publications for presentations and id for the creator you want to insert the presentations into. Let me know if you have any questions :D